### ISSUE TITLE
Are you still working on this?

### ISSUE THREAD

**Issue by Abdul535** [Undisclosed64/Trackr#2]
Hey👋 Tahera,

Are you still working on this? would like to contribute.

**Comment by Undisclosed64** [comment]
Hi, yeah, I'm. Please feel free to improve

**Comment by Abdul535** [comment]
Do you have any todo like, have to this and that?
If yes, please do share.

**Comment by Undisclosed64** [comment]
Yeah, it's not responsive yet. Maybe you can make it responsive on mobile.
